src/backend/drakula/query.sql

```diff
@@ -1,5 +1,4 @@
 CREATE TABLE IF NOT EXISTS games (
     id VARCHAR(255) PRIMARY KEY,
-    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
-    "json" TEXT NOT NULL,
+    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
```

Comment:
Is any part of this actually storing the data?

Reply:
I didn't solve this issue, I delete that query at first PR, because after testing, I didn't find why it didn't store data. I am currently working on it. it should be,  I will find out.